.github/workflows/bundled-lint-and-test.yaml

```diff
@@ -63,9 +63,12 @@ jobs:
           rustup show
           rustup component add rustfmt clippy
 
-      - name: Build the project
+      - name: Build the project without Zenoh & vsomeip streamer reference
```

Comment:
wouldn't that rather be
> without Zenoh & vsomeip transport reference

Actually, it seems like you only moved the dependency to the vsomeip trnasport behind a feature ...

Reply:
Yeah, I'm reconsidering how to do this. Maybe one feature flag per transport. And then if both are enabled we build the streamer references as well.

Reply:
Resolved this comment, looking for more feedback!